Plugins/SequencePlugin/Source/SequencePlugin/Public/Marketplace/Structs/SeqCollectiblesFilter.h

```diff
@@ -0,0 +1,172 @@
+            if (InAccounts.Num() > 0)
+            {
+                ret.Append(",\"inAccounts\":");
+                ret.Append(USequenceSupport::StringListToSimpleString(InAccounts));
+            }
+
+            // Add not in accounts
+            if (NotInAccounts.Num() > 0)
+            {
+                ret.Append(",\"notInAccounts\":");
+                ret.Append(USequenceSupport::StringListToSimpleString(NotInAccounts));
+            }
+
+            // Add orders created by
+            if (OrdersCreatedBy.Num() > 0)
+            {
+                ret.Append(",\"ordersCreatedBy\":");
+                ret.Append(USequenceSupport::StringListToSimpleString(OrdersCreatedBy));
+            }
+
+            // Add orders not created by
+            if (OrdersNotCreatedBy.Num() > 0)
+            {
+                ret.Append(",\"ordersNotCreatedBy\":");
+                ret.Append(USequenceSupport::StringListToSimpleString(OrdersNotCreatedBy));
+            }
+
+            ret.Append("}");
+        }
+        return ret;
+    }
+
+    TSharedPtr<FJsonObject> GetJson()
+    {
+        TSharedPtr<FJsonObject> ret = MakeShareable<FJsonObject>(new FJsonObject);
+        ret->SetBoolField("includeEmpty", bIncludeEmpty);
+        ret->SetStringField("searchText", SearchText);
+
+        if (Properties.Num() > 0)
+        {
+            TArray<TSharedPtr<FJsonValue>> propertiesJsonArray;
+            for (const FSeqPropertyFilter& property : Properties)
+            {
+                TSharedPtr<FJsonObject> propertyJson = MakeShareable<FJsonObject>(new FJsonObject);
+                // Assuming FSeqPropertyFilter has a method to convert to JSON
+                propertyJson = property.GetJson(); // Assuming GetJson() exists
+                propertiesJsonArray.Add(MakeShareable(new FJsonValueObject(propertyJson)));
+            }
+            ret->SetArrayField("properties", propertiesJsonArray);
+        }
+
+        if (Marketplaces.Num() > 0)
+        {
+            ret->SetArrayField("marketplaces", USequenceSupport::StringListToJsonArray(Marketplaces));
+        }
+
+        if (InAccounts.Num() > 0)
+        {
+            ret->SetArrayField("inAccounts", USequenceSupport::StringListToJsonArray(InAccounts));
+        }
+
+        if (NotInAccounts.Num() > 0)
+        {
+            ret->SetArrayField("notInAccounts", USequenceSupport::StringListToJsonArray(NotInAccounts));
+        }
+
+        if (OrdersCreatedBy.Num() > 0)
+        {
+            ret->SetArrayField("ordersCreatedBy", USequenceSupport::StringListToJsonArray(OrdersCreatedBy));
+        }
+
+        if (OrdersNotCreatedBy.Num() > 0)
+        {
+            ret->SetArrayField("ordersNotCreatedBy", USequenceSupport::StringListToJsonArray(OrdersNotCreatedBy));
+        }
+
+        return ret;
+    }
+
+    void Setup(FJsonObject json_in)
+    {
+        // Implement parsing from JSON here if needed
+        // Example: bIncludeEmpty = json_in->GetBoolField("includeEmpty");
```

Comment:
Is this needed? I'm confused by these comments

Reply:
removed, was not needed

Reply:
Can we maybe leave a note that this isn't needed in this case?

Reply:
@BellringerQuinn, if they've been removed, is there any need of a note?